composer.json

```diff
@@ -1,12 +1,12 @@
 {
     "name": "lmc/coding-standard",
-    "description": "Coding standard used in LMC projects",
+    "description": "Coding standard used in Alma Career Czechia (LMC) projects",
     "license": "MIT",
     "type": "library",
     "authors": [
         {
-            "name": "LMC s.r.o.",
-            "homepage": "https://github.com/lmc-eu"
+            "name": "Alma Career",
```

Comment:
```suggestion
            "name": "Alma Career Czechia",
```

Reply:
This refers to the name of the organization, which is Alma Career... I would keep this one as is (but I changed the README references as suggested).